ui/src/App.js

```diff
@@ -53,56 +90,257 @@ function App() {
     console.log("[INFO]: Setup listener.");
     listen('authorize', (event) => {
       console.log("[INFO]: Wake: ", event);
-      setAuthorizationSummary(event.payload);
+      if (event.payload === GET_RECOVERY_PHRASE) {
+        console.log("[INFO]: Wake: ", event);
+        setCurrentPage(EXPORT_RECOVERY_PHRASE_PAGE);
+        appWindow.show();
+        return;
+      }
```

Comment:
Now that we are listening for other things besides tx authorization requests, probably want to rename this function. Also might want to factor out sub-functions for all cases (SEND, WITHDRAW, etc)